Automattic/course
# Update image name and eliminate use of hardcoded URL

This PR renames the image asset and also updates the URL to use `get_template_directory_uri` instead.

## Testing Instructions
- Create a new page.
- Navigate to Sensei LMS patterns.
- Ensure an image is visible in the Testimonials pattern.
- Add the pattern.
- Ensure the image is visible.
